src/core/ext/xds/xds_certificate_provider.cc

```diff
@@ -379,7 +377,9 @@ void XdsCertificateProviderArgDestroy(void* p) {
   xds_certificate_provider->Unref();
 }
 
-int XdsCertificateProviderArgCmp(void* p, void* q) { return GPR_ICMP(p, q); }
+int XdsCertificateProviderArgCmp(void* p, void* q) {
+  return grpc_core::icmp(p, q);
```

Comment:
No need for `grpc_core::` prefix anywhere in the xds tree either.

Reply:
done